thyrel-web/src/components/room/lobby/ShareRoomButton.tsx

```diff
@@ -17,11 +37,22 @@ export default function ShareRoomButton({ identifier }: ShareRoomButtonProps) {
     }
   }
   return (
-    <Button
-      startIcon={<VpnKeyIcon style={{ fontSize: 32 }} />}
-      onClick={onShared}
-      variant="contained">
-      {identifier || 'loading...'}
-    </Button>
+    <Box>
+      <Button
+        startIcon={<VpnKeyIcon style={{ fontSize: 32 }} />}
+        onClick={onShared}
+        variant="contained">
+        {identifier || 'loading...'}
+      </Button>
+      {player?.isOwner && (
```

Comment:
tu peux aussi ajouté 
```suggestion
      {player?.isOwner && identifier && (
```

pour que sa s'affiche que si y'a l'identifier